dotcom-rendering/src/components/RichLink.stories.tsx

```diff
@@ -56,590 +58,796 @@ const mockFormatOpinion = {
 	theme: Pillar.Opinion,
 };
 
-export const Article = () => {
-	return (
-		<Section
-			showTopBorder={false}
-			centralBorder="full"
-			showSideBorders={false}
-		>
-			<Figure format={mockFormatNews} isMainMedia={false} role="richLink">
-				<RichLink
-					richLinkIndex={1}
-					cardStyle="news"
-					imageData={someImageData}
-					headlineText="Rich link headline"
-					contentType="article"
-					url=""
-					linkFormat={mockFormatCulture}
-					format={mockFormatCulture}
-					tags={[]}
-					sponsorName=""
-				/>
-			</Figure>
-		</Section>
-	);
-};
-Article.storyName = 'Article';
-Article.decorators = [splitTheme([mockFormatNews])];
+export const Article = {
+	args: {
+		richLinkIndex: 1,
+		cardStyle: 'news',
+		imageData: someImageData,
+		headlineText: 'Rich link headline',
+		contentType: 'article',
+		url: '',
+		linkFormat: mockFormatCulture,
+		format: mockFormatCulture,
+		tags: [],
+		sponsorName: '',
+	},
+	render: (args) => {
+		return (
+			<SectionComponent
+				showTopBorder={false}
+				centralBorder="full"
+				showSideBorders={false}
+			>
+				<Figure
+					format={args.format}
+					isMainMedia={false}
+					role="richLink"
+				>
+					<RichLink {...args} />
+				</Figure>
+			</SectionComponent>
+		);
+	},
+	parameters: {
+		chromatic: {
+			modes: { horizontal: allModes.splitHorizontal },
+		},
+	},
+} satisfies Story;
 
-export const Network = () => {
-	return (
-		<Section
-			showTopBorder={false}
-			centralBorder="full"
-			showSideBorders={false}
-		>
-			<Figure format={mockFormatNews} isMainMedia={false} role="richLink">
-				<RichLink
-					richLinkIndex={1}
-					cardStyle="special-report"
-					imageData={someImageData}
-					headlineText="Rich link headline"
-					contentType="network"
-					url=""
-					linkFormat={mockFormatCulture}
-					format={mockFormatCulture}
-					tags={[]}
-					sponsorName=""
-				/>
-			</Figure>
-		</Section>
-	);
-};
-Network.story = {
+export const Network = {
+	args: {
+		richLinkIndex: 1,
+		cardStyle: 'special-report',
+		imageData: someImageData,
+		headlineText: 'Rich link headline',
+		contentType: 'network',
+		url: '',
+		linkFormat: mockFormatCulture,
+		format: mockFormatCulture,
+		tags: [],
+		sponsorName: '',
+	},
+	render: (args) => {
+		return (
+			<SectionComponent
+				showTopBorder={false}
+				centralBorder="full"
+				showSideBorders={false}
+			>
+				<Figure
+					format={mockFormatNews}
+					isMainMedia={false}
+					role="richLink"
+				>
+					<RichLink {...args} />
+				</Figure>
+			</SectionComponent>
+		);
+	},
 	parameters: {
 		viewport: { defaultViewport: 'mobileMedium' },
-		chromatic: { viewports: [380] },
+		chromatic: {
+			viewports: [380],
```

Comment:
`modes` is a replacement for `viewports`[^1]. If you have a particular viewport you want to snapshot, you can add it to the list of modes.

```suggestion
```

[^1]: https://www.chromatic.com/docs/viewports/#migration-from-viewports-legacy-to-modes